packages/sysutils/busybox/scripts/fs-resize

```diff
@@ -39,19 +39,20 @@ if [ -e /storage/.please_resize_me ] ; then
   sync
 
   # just in case
-  if [ ! -z "$DISK" -a ! -z "$PART" ] ; then
-    # get storage partition start
-    PART_START=$(parted -s -m $DISK unit b print |grep -v ^/dev |grep -v BYT | grep ^2: |  cut -f2 -d ":")
-
-    # failed to get partition start offset ?
-    if [ ! -z "$PART_START" ] ; then
+  if [ ! -z "$PART_START" ] ; then
```

Comment:
Value of PART_START is 0 and no resize is made.

And btw: LibreELEC 7.0 will not be updated anymore. Only addons.